labellab_mobile/lib/screen/issue/details/issue_details_state.dart

```diff
@@ -0,0 +1,20 @@
+import '../../../model/issue.dart';
+
+class IssueDetailState {
+  bool isLoading = false;
+
+  String? error;
+  Issue? issue;
+
+
+  IssueDetailState.loading({this.issue}) {
```

Comment:
Why are we passing the issue as argument here. Not necessary. We only have to toggle the loading state here.